### DISCUSSION TITLE
bounding box networks

### DISCUSSION THREAD

**Discussion by anais2390** [Project-MONAI/MONAI#3516] [Q&A] [answered]
Hi everyone, 
Sorry for noob question but I am new on the subject : is there currently a monai network avaible for bounding box training task ?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @anais2390 ,

I think maybe you mean detection network?
We don't have detection features so far, but we are already working on it, plan to add in MONAI v1.0 release.

Thanks.

**Reply by anais2390** [reply]
Thanks @Nic-Ma  !
